helm/examples/azure/values-auto-azurefile.yaml

```diff
@@ -8,36 +8,31 @@ storage:
     storageSize: 1Gi
     storageClass: "azurefile"
     prefix: "/var/lib/HPCCSystems/dalistorage"
-    labels:
-    - dali
+    category: dali
```

Comment:
tirival/not new: we're not consistent e.g. here vs other examples and stock values.yaml with how we quote strings, i.e. other places have "dali"

Reply:
I prefer without in this case because it is effectively an enumeration.